apps/client/src/app/export/export.component.ts

```diff
@@ -198,18 +198,33 @@ export class ExportComponent implements OnInit {
     this.cookieService.put(Globals.cookieKeys.exportLastImplementationGuideId + '_' + this.configService.fhirServer, this.options.implementationGuideId);
 
     const igName = this.selectedImplementationGuide.name.replace(/\s/g, '_');
-    const extension = (this.options.responseFormat === 'application/xml' ? '.xml' : '.json');
+
+    const extension = this.options.responseFormat === 'application/xml' ? '.xml' : this.options.responseFormat === 'application/msword' ? '.docx' : '.json';
+
+
+    console.log("*****EXTENSION:" + extension);
 
     try {
       switch (this.options.exportFormat) {
         case ExportFormats.HTML:
-          this.exportService.exportHtml(this.options)
+          //Need to discuss the difference between HTML vs Bundle
```

Comment:
There should be a new enum option on ExportFormats for "MSWord". Need a separate "case" block here.

Reply:
Completed